src/stdstm32-spi.h

```diff
@@ -517,6 +517,10 @@ LL_SPI_InitTypeDef SPI_InitStruct = {};
   LL_GPIO_AF_Remap_SWJ_NOJTAG();
 #endif
 
+#if defined SPI_USE_SPI1_PB3PB4PB5 && defined STM32F1
```

Comment:
this looks great, but could I ask you for a favor and rearrange the defines into
#ifdef STM32F1
#ifdef SPI_USE_SPPB3PB4PB5
  LL....
#endif
#ifdef SPI_USE_SPI3
  LL...
#endif
#endif

it's just to bundle them to make clear they all are STM32 specific

MOST crucial, you cannot do this here. You MUST do this in stdstm32-spi-template.h, and then regenerate the header files by running the respective py script!!

Reply:
Both items should be addressed.